src/Propel/Generator/Command/InitCommand.php

```diff
@@ -411,12 +410,8 @@ private function reverseEngineerSchema(OutputInterface $output, array $options)
      */
     protected function createConsoleHelper(InputInterface $input, OutputInterface $output)
     {
-        /* Check if it runs in Symfony3 env — than use QuestionHelper, because DialogHelper is absent */
-        if (class_exists('\Symfony\Component\Console\Helper\QuestionHelper')) {
-            $helper = new ConsoleHelper3($input, $output);
-        } else {
-            $helper = new ConsoleHelper($input, $output);
-        }
+        $helper = new ConsoleHelper3($input, $output);
```

Comment:
shouldn't we have the ConsoleHelper used, why "3"?

Reply:
The ConsoleHelper was here to support Symfony 2.x (it uses DialogHelper which was removed in 3.0).

IMHO we should remove all the ConsoleHelper stuff and use Symfony helpers.